packages/config/config/devices/0x0403/shha10000.json

```diff
@@ -16,6 +16,240 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"paramInformation": {
+		"1": {
+			"label": "Over current protection",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"2": {
+			"label": "Device Status After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "State before failure",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Off",
+					"value": 2
+				}
+			]
+		},
+		"3": {
+			"label": "Notification Load Status Change",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Send report",
+					"value": 1
+				},
+				{
+					"label": "Send report only with manual change",
+					"value": 2
+				}
+			]
+		},
+		"4": {
+			"label": "LED Indicator Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED is permanently on when device on",
+					"value": 0
+				},
+				{
+					"label": "LED is on for 5 seconds when device changing to on",
+					"value": 1
+				}
+			]
+		},
+		"16": {
+			"label": "Threshold Power Report Value",
+			"description": "The threshold value serves as an additional condition of the time-based report",
+			"valueSize": 2,
+			"unit": "Watt",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 50,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"17": {
+			"label": "Threshold Power Report Percentage",
+			"description": "The threshold value serves as an additional condition of the time-based report",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 127,
```

Comment:
127%, interesting 🤔

Reply:
The manual says.. quite uncommonm but I could find any other info..
![grafik](https://user-images.githubusercontent.com/49494924/109180476-351a2f00-778b-11eb-9d03-86ed4d4250f4.png)

Reply:
Its the same in the english manual..

> -128 –127 = Available Settings

So I would say it is ok even if it is a bit weird.

Reply:
WTF. Well, then it is okay.